packages/components/src/docs/Migration/ui-image.mdx

```diff
@@ -0,0 +1,67 @@
+# Migration Guide: `ui-image`
+
+The new `sd-media` and `sd-copyright` are designed to replace the `ui-image`. Rather than offering a complex component structure, they primarily focus on providing CSS styles.
+
+Media no longer requires the `ui-image` wrapper and instead can directly utilize the provided CSS classes.
+
+## 🎨 CSS Classes
+
+### ✨ New CSS Classes
+
+#### [sd-media]
+
+The `sd-media` class is used to apply the basic styles to the media elements such as margin and default text color.
+
+#### [sd-copyright]
+
+The `sd-copyright` class is used to apply the basic styles to the copyright elements. It positions the element and defines padding, font size, color and text shadow.
+
+<hr />
+
+## ⚙️ Attributes
+
+### ❌ Removed Attributes
+
+Since the new [sd-media] and [sd-copyright] are no longer components basically all attributes have been removed:
+
+1. [altText]
+2. [componentWidth]
+3. [copyrightPosition]
+4. [copyrightText]
+5. [lazyLoading]
+6. [lightbox]
+7. [lightboxBackground]
+8. [maskPosition]
+9. [masked]
+10. [mouseoverText]
+11. [portrait]
+12. [positionX]
+13. [positionY]
+14. [ratio]
+15. [replace]
+16. [sizes]
+17. [src]
+18. [subText]
+19. [subTextHeadline]
+
+<hr />
+
+## 🥳 Events
+
+### ❌ Removed Events:
+
+The following events have been removed from the new components:
+
+1. [componentResized]
+
+<hr />
+
+## 🧪 Methods
+
+### ❌ Removed Methods:
+
```

Comment:
I think we could remove the list of individual attributes since it doesn't make sense to have attributes in a CSS style in the first place.

We decided to not include these for pattern migration guides for example ([ui-suppernumber](https://solid-design-system.fe.union-investment.de/x.x.x/storybook/?path=/docs/docs-migration-ui-supernumber--docs)) I think it makes sense to do the same here.

Reply:
I can see your point but this would create too many different guides, since we already used this structure for other CSS-styles migration guides.

Reply:
I think both of you are right. I would follow our current approach, but at least have the concerns in mind, especially if complains are coming in on that. I'm hesitating doing a ticket for that, as it's maybe not worth the effort. But I would say that's the decision of @karlbaumhauer and @yoezlem

Reply:
I see. Considering the other CSS migration guides did include it, this definitely makes sense. Resolving this conversation to unblock this PR.

Reply:
From my point of view the removed attributes have a value here. 
1st, we removed them and it might help in one or another case to already provide a technical hint on what to use in return here (eg. a variant) and 2nd, we keep the guides in a similar style and they will be easier to read. 

It might (as 3rd point) even help users to summarize the functionality of the component and tell them exactly what they have to implement on their side from now on.